--- a/src/include/common/file_system/file_system.h
+++ b/src/include/common/file_system/file_system.h
@@ -60,10 +60,10 @@
    };
    
    class FileSystem {
-    public:
-        static std::string getFileExtension(const std::filesystem::path& path);
-        static std::string getFileName(const std::filesystem::path& path);
-        static std::string getDirName(const std::filesystem::path& path);
+    public:
+        static std::string getFileExtension(const std::string& path);
+        static std::string getFileName(const std::string& path);
+        static std::string getDirName(const std::string& path);
    };
